src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,57 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
 public class SalaryInfo {
+    private static final int NAME_OF_PERSON = 1;
+    private static final int DATA = 0;
+    private static final int WORKING_HOUR = 2;
+    private static final int INCOME_PER_HOUR = 3;
+    private static final String SEPARATOR = " - ";
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder result = new StringBuilder("Report for period "
+                + dateFrom + SEPARATOR + dateTo + System.lineSeparator());
```

Comment:
StringBuilder is used to avoid creating new String during the concatenation, so use append() method